subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/locking/LockingInteractionsIntegrationTest.groovy

```diff
@@ -264,4 +264,51 @@ dependencies {
         'latest.integration' | '1.2-SNAPSHOT'
     }
 
+    def 'locking and checkstyle integration test'() {
+        file("src/main/java/org/app/App.java") << """
+package org.app;
+
+public class App {
+    public static void main(String[] args) {
+        System.out.println("Hello world");
+    }
+}"""
+
+        file("config/checkstyle/checkstyle.xml") << """
+<!DOCTYPE module PUBLIC
+    "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
+    "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">
+<module name="Checker">
+  <property name="charset" value="UTF-8"/>
+
+  <!-- No tabs -->
+  <module name="FileTabCharacter"/>
+</module>
+"""
+
+        buildFile << """
+apply plugin: 'java'
+apply plugin: 'checkstyle'
+
+repositories {
+    mavenCentral()
```

Comment:
:x: Should use `${${mavenCentralRepository()}}` instead of:

```
repositories {
   mavenCentral()
}
```

Reply:
Fixed